docs/products/kafka/kafka-connect/concepts.rst

```diff
@@ -0,0 +1,6 @@
+Concepts
+========
+
+Explanations and supporting concepts to help you succeed with Aiven for Apache Kafka Connect.
+
+.. tableofcontents::
```

Comment:
This will use proper anchors in the future? Now all links bring to the same top of the page...

Reply:
This is standard across all pages, not sure what we should do differently, maybe a topic for #devportal ?